spec/requests/custom_reactions_controller_spec.rb

```diff
@@ -155,6 +155,21 @@
     end
   end
 
+  context 'changing discourse_reactions_reaction_for_like' do
```

Comment:
this test still doesn’t make it clear what's happening, do a before/after, eg:

initial situation assert X likes Y laughing

change setting

assert X likes and Y laughing